src/storage/store/node_group.cpp

```diff
@@ -128,7 +129,7 @@ void NodeGroup::initializeScanState(Transaction*, const UniqLock& lock,
     TableScanState& state) const {
     auto& nodeGroupScanState = *state.nodeGroupScanState;
     nodeGroupScanState.chunkedGroupIdx = 0;
-    nodeGroupScanState.numScannedRows = 0;
+    nodeGroupScanState.nextRowToScan = 0;
```

Comment:
Ditto about reverting this or getting @ray6080 to review this.

Reply:
Ditto, from @benjaminwinger